### DISCUSSION TITLE
Holidays for Singapore, Philippines and Malaysia

### DISCUSSION THREAD

**Discussion by nasi-lemak** [openpotato/openholidaysapi.website#29] [Ideas]
I would like to contribute to holidays in Singapore, Philippines and Malaysia (including regional holidays for Malaysia).

**Comment by nasi-lemak** [comment]
Currently the public holidays for Singapore for the year 2024 and 2025 can be retrieved from https://www.mom.gov.sg/employment-practices/public-holidays . As for Philippines, there are there are national holidays, non-working holidays and working holidays, only national holidays and non-working holidays are given remuneration if an employee works on that day. The list of holidays for Philippines can be obtained here: https://www.officialgazette.gov.ph/nationwide-holidays .

**Comment by fstueber** [comment]
Could you provide a pull reqeust? Or just a plain csv file with dates and holiday names (native trabnslation would be great) attached to this dicussion?

**Reply by nasi-lemak** [reply]
Sure :D . I'll provide a pull request once I'm done creating the csv files.

**Comment by fstueber** [comment]
Sounds great.
